docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:14.04
+MAINTAINER svanoort <samvanoort@gmail.com>
```

Comment:
@oleg-nenashev Is this a hard blocker for merge?  (I.E. if we merged and then later removed/changed the maintainer, would it be a problem?)  It would be good to have this merged back into master so people can start to consume it.